<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">SimpleTagger</string>
  </property>
  <widget class="QWidget" name="widgetCentral">
   <layout class="QHBoxLayout" name="horizontalLayout_12"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <widget class="QMenu" name="menuRecentProjects">
     <property name="title">
      <string>&amp;Recent projects</string>
     </property>
    </widget>
    <addaction name="actionCreateProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="menuRecentProjects"/>
    <addaction name="actionProjectSetup"/>
    <addaction name="actionCloseProject"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Applicatio&amp;n</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="menuProject"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreateProject">
   <property name="text">
    <string>&amp;Create new project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>&amp;Open project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionProjectSetup">
   <property name="text">
    <string>&amp;Project setup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="text">
    <string>C&amp;lose project</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/bx-info-circle.svg</normaloff>:/icons/bx-info-circle.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
